src/EditorFeatures/Core/Implementation/Organizing/OrganizeDocumentCommandHandler.cs

```diff
@@ -20,7 +20,8 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Organizing
 {
     [ExportCommandHandler(PredefinedCommandHandlerNames.OrganizeDocument,
         ContentTypeNames.CSharpContentType,
-        ContentTypeNames.VisualBasicContentType)]
+        ContentTypeNames.VisualBasicContentType,
+        ContentTypeNames.XamlContentType)]
```

Comment:
I know that C#, VB, and XAML support Organize Document but I'm not so sure about the other languages. I wouldn't want a command to suddenly show up that may cause issues.